build.sbt

```diff
@@ -109,6 +111,8 @@ lazy val chiselFrontend = (project in file("chiselFrontend")).
   ).
   dependsOn(coreMacros)
 
+// Hack to avoid circular dependency on chisel.
+val root = Project("chisel", file("."))
 
 lazy val chisel = (project in file(".")).
```

Comment:
The updated definition (using `RootProject`) should work without requiring any re-definition on the part of the root project. It should always refer to the actual "root" project.